tests/Elasticsearch/Tests/ClientIntegrationTests.php

```diff
@@ -18,9 +19,29 @@
  */
 class ClientIntegrationTests extends \PHPUnit\Framework\TestCase
 {
+
+    /**
+     * @var $client Elasticsearch\Client
+     */
+    protected $client;
+
+    public function setUp()
+    {
+        $this->client = Elasticsearch\ClientBuilder::create()->setHosts([$_SERVER['ES_TEST_HOST']])->build();
+    }
+
+    public function tearDown()
+    {
+        // remove all previously indices created by test or by the before setup
+        $param = [
+            'index' => '_all'
+        ];
+        $this->client->indices()->delete($param);
+    }
+
     public function testCustomQueryParams()
     {
-        $client = Elasticsearch\ClientBuilder::create()->setHosts([$_SERVER['ES_TEST_HOST']])->build();
+        $this->client = Elasticsearch\ClientBuilder::create()->setHosts([$_SERVER['ES_TEST_HOST']])->build();
```

Comment:
I think this line can be removed completely, because `$this->client` is already initialized in `setUp()`

Reply:
luckily you're here, I completely forgot to remove it. Thanks again